GraphQL: Update product query output and add cms_block attribute (#5751)
magento/devdocs: guides/v2.3/graphql/queries/category-list.md

## Output attributes {#Categories}

> The query returns a `CategoryTree` object, which implements [`CategoryInterface`]({{page.baseurl}}/graphql/product/category-interface.html). The `CategoryTree` object can contain the following attribute, as we as all attributes defined in `CategoryInterface`:

Reviewer: ```suggestion
The query returns a `CategoryTree` object, which implements [`CategoryInterface`]({{page.baseurl}}/graphql/product/category-interface.html). The `CategoryTree` object can contain the following attribute, and all attributes defined in `CategoryInterface`:
```
